dashboard: smoother events creation (fixes #9109)

emmanuelbaa opened the pull request:
fixes #9109 

added a time picker to the add event dialogue.

<img width="1918" height="895" alt="timePicker" src="https://github.com/user-attachments/assets/1d47befb-9e57-44ae-9e3b-953aab1b980f" />

emmanuelbaa committed:
added supporting time picker to add event (fixes #9109)

jessewashburn reviewed (changes requested):
I like the idea, but currently I'm seeing some weird margins. 

On Firefox:
<img width="1220" height="976" alt="image" src="https://github.com/user-attachments/assets/ed7b4b38-9809-4940-9f8e-28fc950b4644" />

On Chrome the issue is more subtle, but you can see the top of the 0 and the 2 are slightly cut off. 
<img width="739" height="143" alt="image" src="https://github.com/user-attachments/assets/c0519f2a-776f-4b69-b6fa-30d02bfa03b4" />

Saby-Bishops commented:
Hi @emmanuelbaa,
I think double-checking the CSS for padding/line-height on the input field might be a good idea? It might just need a small adjustment to ensure consistent rendering across browsers.
Otherwise, the functionality looks solid.

google-labs-jules[bot] committed:
fix: correct time picker margins and padding

This change fixes a visual bug in the time picker input fields where the text was being cut off due to incorrect margins and padding. The fix adds a new CSS class to the time picker fields and applies a padding to the input element to ensure the text is displayed correctly in all browsers.

google-labs-jules[bot] committed:
fix: correct time picker margins and padding

This change fixes a visual bug in the time picker input fields where the text was being cut off due to incorrect margins and padding. The fix adds a new CSS class to the time picker fields and applies a height to the input element to ensure the text is displayed correctly in all browsers.

emmanuelbaa committed:
Merge branch 'master' into 9109-addevent-addaSupoortingTimePicker-Baah

emmanuelbaa committed:
Merge branch '9109-addevent-addaSupoortingTimePicker-Baah' of https://github.com/open-learning-exchange/planet into 9109-addevent-addaSupoortingTimePicker-Baah

dogi committed:
Update README.md
